modules/cert-manager-acme-http01.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-http01_{context}"]
+// = Using the HTTP01 Ingress solver
+
+= Issuing certificates by using the HTTP01 challenge
+// TODO: Name
+
+TODO: intro
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* TODO
+// * User workload to expose
+// TODO: ^ from gdoc, what does this mean?
+
+.Procedure
+
+. Create an ACME cluster issuer.
+
+.. Create a YAML file that defines the `ClusterIssuer` object:
++
+.Example `acme-cluster-issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod                                           <1>
+spec:
+  acme:
+    preferredChain: ""
+    privateKeySecretRef:
+      name: example-issuer-account-key-prod                        <2>
+    server: https://acme-staging-v02.api.letsencrypt.org/directory <3>
+    solvers:
+      - http01:
+          ingress:
+            class: openshift-default                               <4>
+----
+<1> TODO: Need description or know if they need to change.
+<2> TODO: Need description or know if they need to change.
+<3> TODO: Need description or know if they need to change.
+<4> TODO - Specify `openshift-default` to use the Ingress-to-Route controller (for what purpose).
```

Comment:
@thejasn @swghosh I did read that Google doc you provided, but couldn't really figure out what to say for why they need to do this. 

Is it REQUIRED that they set this (would it not work if they didn't)? If so, I want to make sure to word it that way.

Reply:
Ingress-to-Route controller is how openshift implements the [Ingress API](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) from upstream k8s.

Reply:
@thejasn Thanks. So is it absolutely required that they specify this, or this wouldn't work?